bindings/dart/test/mobile/utils.dart

```diff
@@ -6,25 +6,13 @@ import 'package:xayn_ai_ffi_dart/src/common/data/history.dart'
     show UserFeedback, History, Relevance, DayOfWeek, UserAction;
 import 'package:xayn_ai_ffi_dart/src/common/result/error.dart'
     show Code, XaynAiException;
-import 'package:xayn_ai_ffi_dart/src/common/reranker/data_provider.dart'
-    show AssetType;
 import 'package:xayn_ai_ffi_dart/src/mobile/reranker/data_provider.dart'
-    show SetupData;
+    show getAssets, SetupData;
 
-const smbertVocab = '../../data/smbert_v0000/vocab.txt';
-const smbertModel = '../../data/smbert_v0000/smbert.onnx';
-const qambertVocab = '../../data/qambert_v0001/vocab.txt';
-const qambertModel = '../../data/qambert_v0001/qambert.onnx';
-const ltrModel = '../../data/ltr_v0000/ltr.binparams';
-
-SetupData mkSetupData(String smbertVocab, String smbertModel,
-    String qambertVocab, String qambertModel, String ltrModel) {
-  return SetupData(<AssetType, String>{
-    AssetType.smbertVocab: smbertVocab,
-    AssetType.smbertModel: smbertModel,
-    AssetType.qambertVocab: qambertVocab,
-    AssetType.qambertModel: qambertModel,
-    AssetType.ltrModel: ltrModel,
+SetupData mkSetupData() {
+  return SetupData({
+    for (final asset in getAssets().entries)
+      asset.key: '../../data/' + asset.value.suffix
```

Comment:
```suggestion
      asset.key: '../../' + asset.value.path
```

Instead of `../../` we could try to detect the workspace root, through it's potentially not worth the amount or work it implies.

Reply:
the dart `Asset` doesn't have a `path` field but only a `suffix` field (in contrast to the `json` file), hence the suggestion won't work.

Reply:
This is a bit troublesome because the whole idea behind `path` is to represent the path relative to the workspace root to the different assets, while the suffix is mobile on mobile specific and could theoretically diverge from path.

I'm wondering if it could make sense to add `path` to `Asset` :thinking: 

I mean the overhead this causes if complete negligible.

Or we somehow for tests set `suffix` = `path`, or something similar??

Reply:
Or we go the other way around:

- remove `path` from `assets.json
- force tests to use a "data dir" where assets are placed in by their path, so
  that `{DATA_DIR}/{suffix}` is always valid. 
- depending on the context `DATA_DIR` could be computed (based on the manifest dir) or
  fixed.
  
That probably would be simpler and now encode that fact that path and suffix can not diverge.

Just as an alternative idea.

Reply:
> I'm wondering if it could make sense to add path to Asset 

Not sure about this because we will only use it internally for the tests and it could accidentally be used (instead of the suffix)  by team blue.

The intent of `path` was to only use it in the release ci to find the assets.